lesson_6/map_func.py

```diff
@@ -0,0 +1,16 @@
+nums = [1, 2, 3, 4, 5, 6, 7, 9, 8]
+
+
+def odd_even(num):
+    if num % 2 == 0:
+        return 'even'
+    if num % 2 == 1:
+        return 'odd'
+
+
+parity = list(map(odd_even, nums))
```

Comment:
I think you should communicate with Dmitry - he has the same problem..
You solve it using two passes in the loop, but we can use only one